src/ChipInput.js

```diff
@@ -522,16 +539,16 @@ ChipInput.propTypes = {
   // openOnFocus: PropTypes.bool,
   chipRenderer: PropTypes.func,
   newChipKeyCodes: PropTypes.arrayOf(PropTypes.number),
-  clearOnBlur: PropTypes.bool,
   allowDuplicates: PropTypes.bool,
   fullWidth: PropTypes.bool,
   fullWidthInput: PropTypes.bool,
-  inputRef: PropTypes.func
+  inputRef: PropTypes.func,
+  blurBehavior: PropTypes.oneOf(['clear', 'add', 'none'])
```

Comment:
Maybe we should rename `none` to `ignore`? :thinking: "No" behavior is somewhat misleading, what do you think?